CMakeLists.txt

```diff
@@ -82,8 +82,8 @@ endif()
 
 ##############################################################################################################
 
-if (BUILD_TESTING AND WITH_PROTOBUF)
-    etn_test(${PROJECT_NAME}-test
+if (WITH_PROTOBUF)
+    etn_test_target(${PROJECT_NAME}
```

Comment:
This is targeted for 2.3.1 which do not have the last update of fty-cmake (and we do not want to get them).

Could you just re-work just this part to still use etn-test ?